incubator/command-config/src/main/java/io/aklivity/zilla/runtime/command/config/internal/openapi/http/proxy/OpenApiHttpProxyConfigGenerator.java

```diff
@@ -0,0 +1,519 @@
+            .inject(namespace -> injectTlsClient(namespace))
+            .binding()
+                .name("tcp_client0")
+                .type("tcp")
+                .kind(CLIENT)
+                .options(TcpOptionsConfig::builder)
+                    .host("") // env
+                    .ports(new int[]{0}) // env
+                    .build()
+                .build()
+            .inject(namespace -> injectVaults(namespace, isTlsEnabled))
+            .build();
+    }
+
+    private NamespaceConfigBuilder<NamespaceConfig> injectGuard(
+        NamespaceConfigBuilder<NamespaceConfig> namespace)
+    {
+        if (isJwtEnabled)
+        {
+            namespace
+                .guard()
+                    .name("jwt0")
+                    .type("jwt")
+                    .options(JwtOptionsConfig::builder)
+                        .issuer("") // env
+                        .audience("") // env
+                        .key()
+                            .alg("").kty("").kid("").use("").n("").e("").crv("").x("").y("") // env
+                            .build()
+                        .build()
+                    .build();
+        }
+        return namespace;
+    }
+
+    private NamespaceConfigBuilder<NamespaceConfig> injectTcpServer(
+        NamespaceConfigBuilder<NamespaceConfig> namespace,
+        String plainExit,
+        String tlsExit)
+    {
+        namespace
+            .binding()
+                .name("tcp_server0")
+                .type("tcp")
+                .kind(SERVER)
+                .options(TcpOptionsConfig::builder)
+                    .host("0.0.0.0")
+                    .ports(allPorts)
+                    .build()
+                .inject(binding -> injectTcpRoutes(binding, plainExit, tlsExit))
+                .build();
+        return namespace;
+    }
+
+    private BindingConfigBuilder<NamespaceConfigBuilder<NamespaceConfig>> injectTcpRoutes(
+        BindingConfigBuilder<NamespaceConfigBuilder<NamespaceConfig>> binding,
+        String plainExit,
+        String tlsExit)
+    {
+        if (isPlainEnabled && isTlsEnabled)
+        {
+            binding
+                .route()
+                    .when(TcpConditionConfig::builder)
+                        .ports(httpPorts)
+                        .build()
+                    .exit(plainExit)
+                    .build()
+                .route()
+                    .when(TcpConditionConfig::builder)
+                        .ports(httpsPorts)
+                        .build()
+                    .exit(tlsExit)
+                    .build();
+        }
+        else if (isPlainEnabled)
+        {
+            binding
+                .route()
+                    .exit(plainExit)
+                    .build();
+        }
+        else if (isTlsEnabled)
+        {
+            binding
+                .route()
+                    .exit(tlsExit)
+                    .build();
+        }
+        return binding;
+    }
```

Comment:
Inject these `http` (plain) and `https` (tls) routes using separate methods, which are no-ops if not needed.

Reply:
done